google-cloud-service_directory-v1beta1/proto_docs/google/api/resource.rb

```diff
@@ -0,0 +1,247 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+
+module Google
+  module Api
+    # A simple descriptor of a resource type.
+    #
+    # ResourceDescriptor annotates a resource message (either by means of a
+    # protobuf annotation or use in the service config), and associates the
+    # resource's schema, the resource type, and the pattern of the resource name.
+    #
+    # Example:
+    #
+    #     message Topic \\\{
```

Comment:
Any idea why these backslashes are here?

Reply:
I vaguely remember seeing this before.

Reply:
Hmm. I vaguely remember seeing something like this as well. There's probably some double-escaping going on. I'll take a look at the inputs.

Reply:
This is a tough one, because a fully correct solution would require us to parse markdown. However, we have a better heuristic implemented in synthtool, so I'll copy that over to the generator.

Reply:
@quartzmo Would you be okay with merging this one as-is for now (as we've done for the other microgenerator APIs so far, since they pretty much all exhibit this issue to one degree or another)? I'm working on a generator fix for this, and when it's done, autosynth will regenerate all the libraries with the fix.

Reply:
Updated the escaping algorithm in the microgenerator and regenerated.